INSTALL

```diff
@@ -66,19 +66,29 @@ Check System Prerequisites
     running from a distro many of these are taken care of, if not here they
     are:
 
-     * platform must have one of the following Intel® Communications devices:
-        4xxx  : Use "lspci -d 8086:4940" to check Physical Function (PF)
-                devices are present.
-        401xx : Use "lspci -d 8086:4942" to check PF devices are present.
-        420xx : Use "lspci -d 8086:4946" to check PF devices are present.
+     * platform must have an Intel® Communications device installed.
+        Supported devices:
+        ___________________________
```

Comment:
Can you use `-` instead of `_`? This is what is used for tables

Reply:
addressed